platformio.ini

```diff
@@ -397,18 +376,12 @@ platform      = ststm32
 framework     = arduino
 board         = genericSTM32F103ZE
 extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin.py
-build_flags   = !python Marlin/src/HAL/HAL_STM32F1/STM32F1_flag_script.py
+build_flags   = !python Marlin/src/HAL/HAL_STM32F1/flag_script.py
   ${common.build_flags} -std=gnu++14
 build_unflags = -std=gnu++11
 src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32F1>
 lib_deps      = ${common.lib_deps}
-lib_ignore    = TMC26XStepper
-  libf3c
-  lib066
-  Adafruit NeoPixel_ID28
-  Adafruit NeoPixel
-  libf3e
-  U8glib-HAL
+lib_ignore    = Adafruit NeoPixel, TMC26XStepper
```

Comment:
@tpruvot — Indeed! `U8GLib-HAL` was removed some time ago due to the error you cite.

Reply:
on the lite maybe ? (without fsmc screen)

Reply:
Well, just here in this PR so far.